### ISSUE TITLE
Superseeded by ssb-handshake?

### ISSUE THREAD

**Issue by stv0g** [sbillig/shs_async#1]
Is this repo superseeded by [ssb-handshake](https://github.com/sunrise-choir/ssb-handshake)?

Just wondering :) I am trying to get into the sunrise-choir codebase.

**Comment by sbillig** [comment]
@stv0g Indeed it is! Ideal me intended to update the readme, etc, but actual me rarely gets around to such things. Leaving this open as a reminder.

I'd be happy to discuss the broader codebase verbally and/or textually if you want, and we'd absolutely love a hand if you're inclined to help out. Are you on ssb?

My current short term todos:
- finish/polish message signing/verifying
- standardize key types across different crates
- finish/test prototype muxrpc stuff (mux branch of ssb-packetstream)

**Comment by stv0g** [comment]
Hi Sean,

> Ideal me intended to update the readme, etc, but actual me rarely gets around to such things. Leaving this open as a reminder.

Do you accept pull requests for such minor changes? Or is this actually more work than doing it yourself?

> I'd be happy to discuss the broader codebase verbally and/or textually if you want, and we'd absolutely love a hand if you're inclined to help out. Are you on ssb?

That sounds great :) I've planned to dedicate my winter vacation this year (month of december) to improve my Rust skills and joining the ssb/sunrise-choir development.

You can find me on ssb with my feed `@dArlBpL16fq8eJp49rT/i9+efp5g4qT/XTiFehWYmYY=.ed25519`

> My current short term todos: [...]

I was looking for some small NPM packages which I could port to Rust as a first little task.
Do you have some missing pieces in mind on which I could work?

**Comment by sbillig** [comment]
@stv0g Yes! Any and all pull requests are very welcome; I'm very grateful for the help. I'm traveling at the moment, and about to head into the woods, so I may respond a bit slowly.

I'll get back to you soon about module ideas when I have a bit more time to think and write!
